crates/ruff/src/rules/pandas_vet/rules/inplace_argument.rs

```diff
@@ -8,6 +8,29 @@ use crate::rules::pandas_vet::fixes::fix_inplace_argument;
 use crate::violation::AlwaysAutofixableViolation;
 
 define_violation!(
+    /// ## What it does
+    /// Checks for `inplace=True` in code using the `pandas` library.
+    ///
+    /// ## Why is this bad?
+    /// - `inplace=True` often does not provide a performance benefit. It is
```

Comment:
nit: In the other rule documentations we have been using `*` as a list marker rather than `-` ... it would be nice to be consistent.